components/nifosg/particle.hpp

```diff
@@ -30,11 +30,13 @@ namespace NifOsg
         ParticleSystem();
         ParticleSystem(const ParticleSystem& copy, const osg::CopyOp& copyop);
 
-        META_Object(NifOsg, NifOsg::ParticleSystem)
```

Comment:
Actually it looks like the META_Object() macro determines which namespace is used to load the dynamic serializer library, so these might all need to change to OpenMW.  Let me continue to investigate before changing anything.